libnavigation-ui/src/main/res/values/strings.xml

```diff
@@ -18,8 +18,8 @@
     <!-- Feedback Guidance Issue -->
     <string name="feedback_guidance_issue">Guidance Issue</string>
 
-    <!-- Feedback type for Incorrect Visual -->
-    <string name="feedback_type_incorrect_visual">Incorrect\nVisual</string>
+    <!-- Feedback type for Looks Incorrect -->
+    <string name="feedback_type_looks_incorrect">Looks\nIncorrect</string>
```

Comment:
Noting that this is technically breaking SemVer as this resource is publicly available / accessible. Same for `ic_feedback_incorrect_visual.xml`